### ISSUE TITLE
Towards a Theoretical Understanding of Batch Normalization

### ISSUE THREAD

**Issue by Swall0w** [Swall0w/papers#465]
Kohler, Jonas, Daneshmand, Hadi, Lucchi, Aurelien, Zhou, Ming, Neymeyr, Klaus, Hofmann, Thomas

Normalization techniques such as Batch Normalization have been applied very
successfully for training deep neural networks. Yet, despite its apparent
empirical benefits, the reasons behind the success of Batch Normalization are
mostly hypothetical. We thus aim to provide a more thorough theoretical
understanding from an optimization perspective. Our main contribution towards
this goal is the identification of various problem instances in the realm of
machine learning where, under certain assumptions, Batch Normalization can
provably accelerate optimization with gradient-based methods. We thereby turn
Batch Normalization from an effective practical heuristic into a provably
converging algorithm for these settings. Furthermore, we substantiate our
analysis with empirical evidence that suggests the validity of our theoretical
results in a broader context.

https://arxiv.org/abs/1805.10694
